tests/test_awslambda/test_lambda.py

```diff
@@ -145,46 +147,13 @@ def test_invoke_requestresponse_function(invocation_type):
 
     # Logs should not be returned by default, only when the LogType-param is supplied
     success_result = conn.invoke(
-        FunctionName="testFunction", Payload=json.dumps(in_data), **kw
+        FunctionName=name_or_arn, Payload=json.dumps(in_data), **kw
     )
 
     success_result["StatusCode"].should.equal(200)
     assert "LogResult" not in success_result
 
 
-@pytest.mark.network
-@mock_lambda
-def test_invoke_requestresponse_function_with_arn():
```

Comment:
This test is now duplicated by `test_invoke_requestresponse_function(key=FunctionArn)`